### ISSUE TITLE
Issue with Stripe and resetClasses

### ISSUE THREAD

**Issue by Megafry** [verbb/formie#2092]
### Describe the bug

The Stripe payment doesn't work if `resetClasses` is enabled.
The script is not able to resolve `this.form.getClasses("errorMessage")`  

[payment-provider.js](https://github.com/verbb/formie/blob/craft-5/src/web/assets/frontend/src/js/payments/payment-provider.js#L14)
```js
Uncaught SyntaxError: Failed to execute 'querySelector' on 'Element': '.' is not a valid selector.
```
```js
 removeError() {
   s(this.$field, this.errorClass);
   const e = this.$field.querySelector(`.${this.errorMessageClass}`);
   e && e.remove()
}
```

### Steps to reproduce

1. Import this form - [formie-testForm.json](https://github.com/user-attachments/files/17361692/formie-testForm.json)
2. Set the `resetClasses` to `true`
3. Try to submit the Stripe payment form.
4. The console shows errors

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes 

### Craft CMS version

5.4.4

### Plugin version

3.0.8

### Multi-site?

no

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.8"`.

**Comment by engram-design** [comment]
Fixed in [3.0.9](https://github.com/verbb/formie/releases/tag/3.0.9)
